apertium-tur-uzb.tur-uzb.dix

```diff
@@ -2695,6 +2720,10 @@
     <e><p><l>al<s n="v"/><s n="tv"/></l><r>ol<s n="v"/><s n="tv"/></r></p></e>
     <e><p><l>bitir<s n="v"/><s n="tv"/></l><r>tugat<s n="v"/><s n="tv"/></r></p></e>
     <e><p><l>koy<s n="v"/><s n="tv"/></l><r>qoʻy<s n="v"/><s n="tv"/></r></p></e>
+    <e c="0"><p><l>seyret<s n="v"/><s n="tv"/></l><r>koʻr<s n="v"/><s n="tv"/></r></p></e>
+    <e><p><l>seyret<s n="v"/><s n="tv"/></l><r>tomosha<b/>qil<s n="v"/><s n="tv"/></r></p></e>
+    <e><p><l>say<s n="v"/><s n="tv"/></l><r>sana<s n="v"/><s n="tv"/></r></p></e>
+     <e><p><l>konuş<s n="v"/><s n="iv"/></l><r>gapir<s n="v"/><s n="iv"/></r></p></e>
```

Comment:
"konuş" is transitive

Reply:
I don't think it is.  Could you give an example with an accusative complement?

Reply:
Ex: onu konuştu